script/configs/mainnet/M2_mainnet_upgrade.config.json

```diff
@@ -0,0 +1,46 @@
+{
+  "chainInfo": {
+    "chainId": 1
+  },
+  "multisig_addresses": {
+    "communityMultisig": "0xFEA47018D632A77bA579846c840d5706705Dc598",
+    "executorMultisig": "0x369e6F597e22EaB55fFb173C6d9cD234BD699111",
+    "operationsMultisig": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "pauserMultisig": "0x5050389572f2d220ad927CcbeA0D406831012390",
+    "timelock": "0xA6Db1A8C5a981d1536266D2a393c5F8dDb210EAF"
+  },
+  "strategies": {
+    "numStrategies": 0,
+    "MAX_PER_DEPOSIT": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "MAX_TOTAL_DEPOSITS": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "strategiesToDeploy": []
+  },
+  "strategyManager": {
+    "init_strategy_whitelister": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "init_paused_status": 1
+  },
+  "delegationManager": {
+    "init_paused_status": 0,
+    "init_minWithdrawalDelayBlocks": 50400
+  },
+  "avsDirectory": {
+    "init_paused_status": 0
+  },
+  "slasher": {
+    "init_paused_status": 115792089237316195423570985008687907853269984665640564039457584007913129639935
+  },
+  "eigenPod": {
+    "MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR": 32000000000000000000,
```

Comment:
Is this correct? it looks to be denominated in wei, not gwei

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/480/commits/888d3c6b02a8f0f2261ca7590b0852f3c98cccd0

Reply:
Added a modulo check in case this value gets overflowed again

Reply:
LGTM, thank you